VisualizationsRFiles/T-Tests.R

```diff
@@ -18,4 +20,25 @@ ggplot(Sales, aes(Store, Revenue, label = Revenue, fill = Store)) +
   theme(legend.position="none") + # Gets rid of the legend
   labs(title = "Average Revenue per day Between Store A and Store B", #Adds a title for the plot
      x = NULL, y = "Average Revenue" #Labels for the x and y axis
-     ) 
\ No newline at end of file
+     ) 
+
+
+write.table(Sales, file="Kamal.csv", sep = ',')
```

Comment:
Good idea adding in the data itself. We can make that available for users as a secondary option to recreate plots if they're interested in exploring the coding aspect. Let's keep providing this for the rest of the inferential tests

When you use the `write.csv` function, it produces a csv file that is easier to parse for users than when you use `write.table`. Just something to note moving forward.

```suggestion
write.csv(Sales, #Indicates to write the Sales dataframe
	"filepath/TTestData.csv") #Indicates the filepath to save the data to
	
```

Reply:
understood